.github/workflows/pull.yml

```diff
@@ -570,6 +607,7 @@ jobs:
       docker-image: ${{ needs.linux-focal-py3_12-clang10-experimental-split-build.outputs.docker-image }}
       test-matrix: ${{ needs.linux-focal-py3_12-clang10-experimental-split-build.outputs.test-matrix }}
       timeout-minutes: 600
+    secrets: inherit
```

Comment:
Not sure if `secrets: inherit` is better than passing explicit secrets workflow might need

Reply:
For this to be profitable, there would have to be a case where one instantiation of linux_build.yml needs secrets, but another does not. For the one secret I have in scribe-pr right now, there is no reason to hide it, it should apply uniformly everywhere. So inherit seems like the right tool for the job.